### PR TITLE
feat(nuxt): add `prerenderRoutes` ssr composable

### PR DESCRIPTION
**Pull request by danielroe**
### ❓ Type of change
- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This adds a new composable that will hint to nitro to prerender a path. If we need to change the internal header or make changes, this should make it easier to do so.

Alternatively, we could locate this composable within Nitro auto-imports and re-export in Nuxt so there's at least a stub function on client. Thoughts welcome @pi0.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/3.api/3.utils/prerender-path.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+description: prerenderPath hints to Nitro to prerender an additional route.
+---
+# `prerenderPath`
```
Comment (Atinux) [inline docs/3.api/3.utils/prerender-path.md:4]:
```suggestion
---

# `prerenderPath`
```

**Comment by Atinux** [comment]
For consistency with `nitro.prerender.routes` I would rename it to `prerenderRoute` or `addPrerenderRoute`.

The `add` prefix lets us know it's adding it to a list where I except `prerenderRoute()` to be async and returns me the HTML.

**Comment by danielroe** [comment]
Good idea. Will also refactor to accept an array.

**Comment by danielroe** [comment]
Maybe `queuePrerenderRoute`?

**Comment by pi0** [comment]
I like it! Let's keep it close to Nuxt for now since you might want to make variations in API (but also fits for a Nitro built-in composable PR welcome 👍🏼 )

I personally like `prerenderRoute` since is a straight forward name and does not imply how we do prerender (enqueue or immediate) and is open to changes.

**Comment by Arthur-Chevron** [comment]
> Good idea. Will also refactor to accept an array.

**Comment by Atinux** [comment]
Alright, then let's go fo `prerenderRoute(string)`

**Pull request merged**